html_renderer/src/Frame.vue

```diff
@@ -91,10 +91,16 @@ export default {
       return `${this.frame.filePathShort}:${this.frame.lineNo.toString().padEnd(4, ' ')}`
     },
     formattedTime() {
-      return this.frame.time.toLocaleString(undefined, {
-        minimumFractionDigits: 3,
-        maximumFractionDigits: 3,
-      })
+      if (appState.timeFormat === "absolute") {
+        return this.frame.time.toLocaleString(undefined, {
+          minimumFractionDigits: 3,
+          maximumFractionDigits: 3,
+        });
+      }
+      return `${(this.frame.proportionOfTotal * 100).toLocaleString(undefined, {
+        minimumFractionDigits: 1,
+        maximumFractionDigits: 1,
+      })}%`;
```

Comment:
Maybe nice to firm this up a bit?

```suggestion
      } else if (appState.timeFormat === 'proportion') {
        return `${(this.frame.proportionOfTotal * 100).toLocaleString(undefined, {
          minimumFractionDigits: 1,
          maximumFractionDigits: 1,
        })}%`;
      } else {
        throw new Error("unknown timeFormat")
      }
```

Reply:
Good call, I've pushed the changes!